playwright/page-objects/create-rosa-wizard-page.ts

```
@@ -449,6 +471,22 @@ export class CreateRosaWizardPage extends BasePage {
     await this.domainPrefixInput().blur();
   }
 
+  async closePopoverAndNavigateNext(): Promise<void> {
+    // Popover dialogs keep showing multiple times during next button click
+    // We need to close them all until none are found
+    const maxAttempts = 10;
+    await this.rosaNextButton().click();
+    for (let attempt = 0; attempt < maxAttempts; attempt++) {
```

ESLint

[error] Unary operator '++' used.

(no-plusplus)